.changeset/long-needles-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`STable`): replace comma with underscore to prevent error when import by scss (tailwind style)
```

Comment:
```suggestion
**fix**(`STable`): fix WindiCSS utility usage - replace comma with underscore (~~`grid-cols-[min-content,1fr]`~~ `grid-cols-[min-content_1fr]`). Comma caused an error when compiled styles are imported through SCSS.
```

Looks not bad:

(~~`grid-cols-[min-content,1fr]`~~ `grid-cols-[min-content_1fr]`)

Reply:
changed